azext_edge/edge/providers/support/base.py

```diff
@@ -21,6 +21,8 @@
 logger = get_logger(__name__)
 generic = client.ApiClient()
 
+DAY_IN_SECONDS: int = 60 * 60 * 24
```

Comment:
should prob do this for other pods too - maybe slap a low hanging #TODO comment?